### DISCUSSION TITLE
Issue with Setting Line Number Color

### DISCUSSION THREAD

**Discussion by Armadillidiid** [LazyVim/LazyVim#1892] [Q&A]
I'm facing an issue with setting the line number color in my Neovim configuration. I've added the following line to my `options.lua` file to change the line number color:

```lua
vim.api.nvim_set_hl(0, "LineNr", { fg = "#cdd6f4" })
```

However, it doesn't seem to be taking effect on startup. I have to manually type this command every time I boot up nvim to get it working.

Any insights or suggestions on how to resolve this would be greatly appreciated.

**Comment by dpetka2001** [comment]
Probably a plugin overwrites it. Better to use an autocmd on `BufReadPre/Post` event.

**Reply by folke** [reply]
Any colorscheme would indeed overwrite those hl groups. Colorschemes get loaded after loading options.lua

**Comment by Armadillidiid** [comment]
I got it working with the following. Thank you @dpetka2001  @folke 

```lua
-- Set highlight groups
vim.api.nvim_create_autocmd("BufRead", {
  callback = function()
    vim.api.nvim_set_hl(0, "CursorLineNr", { fg = "#89b4fa" })
  end,
})
```
